stl/inc/thread

```diff
@@ -11,6 +11,7 @@
 #include <chrono>
 #include <memory>
 #include <process.h>
+#include <stop_token>

```

Comment:
This will also emit the "requires C++20" warning for C++14/17 consumers of `<thread>`.